nimble/controller/src/ble_ll_scan.c

```diff
@@ -3081,7 +3081,7 @@ ble_ll_scan_rx_pkt_in_on_aux(uint8_t pdu_type, struct os_mbuf *om,
     if (!BLE_MBUF_HDR_CRC_OK(hdr) ||
         BLE_MBUF_HDR_IGNORED(hdr) ||
         BLE_MBUF_HDR_AUX_INVALID(hdr) ||
-        (aux_data->flags_ll & BLE_LL_AUX_FLAG_SCAN_ERROR) ||
+        (aux_data && aux_data->flags_ll & BLE_LL_AUX_FLAG_SCAN_ERROR) ||
```

Comment:
aux_data cannot be null here, see comment at line 3075.
also please do not mix fixes for LL and host in single PR

Reply:
ok, will remember :) so this is host-only PR now